src/ng/directive/ngOptions.js

```diff
@@ -156,6 +159,9 @@ var ngOptionsMinErr = minErr('ngOptions');
  *     `value` variable (e.g. `value.propertyName`). With this the selection is preserved
  *      even when the options are recreated (e.g. reloaded from the server).
  *
+ * @param {expression=} ngEmptyValue Any empty options, whether hard-coded or
+ * created with ngOptions, will be given the value that this expression evaluates to.
```

Comment:
The empty option is still `''` if I am not mistaken. The ngEmptyValue is passed to $setViewValue and then ends up as the model.